### PR TITLE
Remove `test_data` with in-memory image generation

### LINKED ISSUE #148: Removing test_data and replacing Event unit-test
## Problem Description
Removing the `test_data` subdirectory of `ACM_General`. This file contains a picture which is used to test the event; however, a picture can be automatically generated during test time by python in a temporary directory which will remove the need for this folder.

### Proposed / Possible Solution
Generate a picture during test time for the Event model test in a tempfile

### PR DESCRIPTION
**Pull request by kevinschoonover**
Closes #148.

### CODE REVIEW

File: ACM_General/events/tests.py [hunk 1]
```diff
@@ -2,9 +2,13 @@
 Contains all unit tets for Events app.
 """
 # standard library
-from datetime import datetime
```
Comment (ZachWileman) [inline ACM_General/events/tests.py]: The imports here should be alphabetically organized.

File: ACM_General/rest_api/tests.py [hunk 2]
```diff
@@ -3,8 +3,10 @@
 """
 # standard library
 import json
```
Comment (ZachWileman) [inline ACM_General/rest_api/tests.py]: imports should be alphabetically organized.

**Review by ZachWileman — CHANGES_REQUESTED** [review]
Imports should be alphabetically organized.

**Comment by kevinschoonover** [comment]
Should be fixed now @zdw27f

**Review by ZachWileman — APPROVED** [review]

**Pull request merged**